src/declarations/prosemirror-example-setup.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "prosemirror-keymap" {
```

Comment:
As mentioned on my other PR review, this module name doesn't match the file.

Reply:
👍  that's fixed in #19 so I'll merge and fix forward.